src/hooks/useImageUpload.ts

```diff
@@ -0,0 +1,45 @@
+import AWS from 'aws-sdk';
+import { ChangeEvent } from 'react';
+
+import { IMAGE_EXTENSION, IMAGE_TYPE } from '@/constants';
+
+const useImageUpload = () => {
+  const S3_BUCKET = 'travel-zip-bucket';
+  const s3 = new AWS.S3({ params: { ACL: 'public-read' } });
+
+  AWS.config.update({
+    accessKeyId: process.env.NEXT_PUBLIC_ACCESS_KEY_ID,
+    secretAccessKey: process.env.NEXT_PUBLIC_SECRET_ACCESS_KEY,
+    region: 'ap-northeast-2',
+  });
+
+  const handleFileInput = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files) {
+      const file = e.target.files[0];
+      if (file.size >= 1 * 1024 * 1024) {
+        alert('1mb 이하의 파일만 업로드 가능합니다.');
+        return;
+      }
+      const fileExtension = file.name.split('.').pop() ?? '';
+      if (!IMAGE_TYPE.includes(file.type) || !IMAGE_EXTENSION.includes(fileExtension)) {
+        alert('jpeg/png 파일만 Upload 가능합니다.');
+        return;
+      }
+    }
+  };
+
+  const uploadFile = async (file: File, key: string) => {
+    let location = '';
+    const upload = s3.upload({ Bucket: S3_BUCKET, Body: file, Key: key });
```

Comment:
p1;

```suggestion
    const upload = s3.upload({ 
      Bucket: S3_BUCKET, 
      Body: file, 
      Key: key,
      ContentType: 'image/jpeg',
    });
```

`content-type: octet-stream`을 기본값으로 가지고 있어서 url로 이동하면 다운로드가 되는 현상이 발생합니다. ContentType 옵션을 추가해주세요.